profiles/desc/cpu_flags_arm.desc

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2017 Gentoo Foundation.
+# Distributed under the terms of the GNU General Public License v2
+
+aes - Use the AES cryptography instruction set
+crc32 - Use the CRC32 instruction set
+edsp - Use the enhanced DSP instructions (ARMv*E and ARMv6+)
+iwmmxt - Use the iwMMXt instruction set
+neon - Use the NEON (ASIMD) instruction set
+sha1 - Use the SHA-1 cryptography instruction set
+sha2 - Use the SHA-2 cryptography instruction set
+thumb - Enable Thumb instruction set (ARMv*T and ARMv6+)
+thumb2 - Enable Thumb-2 instruction set (ARMv*T2 and ARMv7+)
+v4 - Use ARMv4 instructions
+v6 - Use ARMv6 instructions
+v7 - Use ARMv7 instructions
+v8 - Use ARMv8 instructions
+vfp - Use the VFP instruction set
+vfp-d16 - Indicate that the FPU has 16 64-bit VFP registers
+vfp-d32 - Indicate that the FPU has 32 64-bit VFP registers
+vfpv3 - Use the VFP version 3 instruction set
```

Comment:
I've used separate flags since that's how the kernel exposes it.

```
#define HWCAP_VFPv3	(1 << 13)
#define HWCAP_VFPv3D16	(1 << 14)	/* also set for VFPv4-D16 */
...
#define HWCAP_VFPv4	(1 << 16)
...
#define HWCAP_VFPD32	(1 << 19)	/* set if VFP has 32 regs (not 16) */
```